crt_portal/cts_forms/templates/forms/complaint_view/show/description.html

```diff
@@ -9,6 +9,7 @@ <h3 class="complaint-card-heading text-uppercase">
       Personal description
       <small class="margin-left-auto">
         <a class="related-reports"
+           onclick="gtag('event', 'click', {'event_name': 'view related reports'});"
```

Comment:
how might we add this to components generated by django (a sufficiently annoying example might be the DJ number filter?)

Reply:
updated to use gaUtil JS file and track clicks by class names and ids